src/multiset.rs

```diff
@@ -327,3 +350,24 @@ impl<A> FromIterator<A> for HashMultiSet<A> where
         multiset
     }
 }
+
+impl<T> PartialEq for HashMultiSet<T>
+where
+    T: Eq + Hash,
+{
+    fn eq(&self, other: &HashMultiSet<T>) -> bool {
+        if self.total_elements() != other.total_elements() {
+            return false;
+        }
+
```

Comment:
Now that #12 is merged, you can also do a quick `.len()` comparison.

Reply:
Is good now.